### PR TITLE
fix: fix Groovy syntax

### PR DESCRIPTION
**Pull request by PetrakovichVictoria**

### CODE REVIEW

File: docs/topics/gradle/gradle-compiler-options.md [hunk 1]
```diff
@@ -78,19 +78,26 @@ It is also possible to configure all of the Kotlin compilation tasks in the proj
...
 </tab>
 <tab title="Groovy" group-key="groovy">
 
 ```groovy
-tasks.withType(org.jetbrains.kotlin.gradle.dsl.KotlinCompile).configureEach {
-    kotlinOptions { /*...*/ }
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompilationTask
+// ...
+
+tasks.named('compileKotlin', KotlinCompilationTask) {
```
Comment (PetrakovichVictoria) [inline docs/topics/gradle/gradle-compiler-options.md:96]: I'm not sure about `named('compileKotlin')`

File: docs/topics/gradle/gradle-compiler-options.md [hunk 2]
```diff
@@ -93,19 +93,26 @@ It is also possible to configure all of the Kotlin compilation tasks in the proj
 <tab title="Kotlin" group-key="kotlin">
 
 ```kotlin
-tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompilationTask<*>>().configureEach {
-    compilerOptions { /*...*/ }
-}
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompilationTask
+// ...
+
+val compileKotlin: KotlinCompilationTask<*> by tasks
```
Comment (Tapchicoma) [inline docs/topics/gradle/gradle-compiler-options.md:99]: Better to use here the same approach as for Groovy:
```
val compileKotlin = tasks.named<KotlinCompilationTask<*>>("compileKotlin")
```

Comment (Tapchicoma) [reply docs/topics/gradle/gradle-compiler-options.md:99]: `by tasks` eagerly realizes task

File: docs/topics/gradle/gradle-compiler-options.md (hunk 1 above)
Comment (Tapchicoma) [reply docs/topics/gradle/gradle-compiler-options.md:111]: `tasks.named('compileKotlin', KotlinCompilationTask)` should work

**Comment by PetrakovichVictoria** [comment]
I merged because I've fixed all @Tapchicoma's remarks. Seems like there is no need to waste his time in rechecking.

**Pull request merged**